src/translations/strings.yaml

```diff
@@ -575,14 +578,18 @@ onboarding:
   privacySlideHeader: Get more privacy
   privacySlideBody: Use these features for more protection. They may cause issues on some sites, so you can turn them off anytime in settings.
   devicesSlideHeader: Install Mozilla VPN on up to 5 devices
-  devicesSlideBody: Scan this QR code using a tablet or mobile device’s camera to install the VPN on another device.
+  devicesSlideBody2: To install the VPN on another tablet or mobile device, scan the QR code.
+  devicesSlideBodyMobile: To install the VPN on another tablet or mobile device, scan the QR code. To install on a desktop computer, visit the Download page on the Mozilla VPN website.
   devicesSlideDeviceTypeLabel: "Select device type:"
   devicesSlideDeviceTypeAndroid: Android
   devicesSlideDeviceTypeApple: Apple
   devicesSlideQRCodeAndroid: QR Code that links to the Mozilla VPN product page on the Google Play Store
   devicesSlideQRCodeApple: QR Code that links to the Mozilla VPN product page on the iOS App Store
   startSlideHeader: Connect Mozilla VPN on startup
   startSlideBody: Turn on to launch the VPN whenever you start your device. We’ll automatically reconnect you to the same server location you last used.
+  startSlideMobileHeader: Allow Mozilla VPN to create a VPN connection
+  startSlideMobileBody: Your mobile device will now ask you to allow us to create a VPN connection. Please allow this so that Mozilla VPN works correctly on your device.
+  startSlideApproveButtonLabel: Approve Mozilla VPN
```

Comment:
Why the switch to approve, if the terminology has been "allow" so far (and probably is in the phone)?

> Your mobile device will now ask you to allow us to create a VPN connection. Please allow this so that Mozilla VPN works correctly on your device.

I find it very confusing when software uses "we" to refer to itself. We're already repeating "allow" a lot, it doesn't seem so bad to repeat "Mozilla VPN".

> Your mobile device will now ask you to allow Mozilla VPN to create a VPN connection. Please allow this so that Mozilla VPN works correctly on your device.

Reply:
agree with all of this! @traceyvantyghem thoughts?

Reply:
Agree about "Allow".

I think we could get away from using "us" here, but maybe we could avoid a little of the repetition. What about this:

**Allow Mozilla VPN to create a connection**

Your mobile device will now ask you to allow Mozilla VPN to create a connection. Please allow this so that the VPN works correctly on your device.

[Connect Mozilla VPN]